travvy-server/src/controllers/AuthenticationController.js

```diff
@@ -0,0 +1,24 @@
+
+
+const Users = require('../models').Users
+
+// This is the logic for adding users to the database
+// The format is set to match the Users table on MySQL
+// The null values will be replaced with information that the user will provide during onboarding
+module.exports = {
+  async signup (email,name, password) {
+    const user = await Users.create({
```

Comment:
Changed to  "users" for now to reflect the database even though it is one user. Need to keep in mind to stay consistent in the future